examples/tests/invalid/parameters.test.w

```diff
@@ -0,0 +1,18 @@
+let app = nodeof(this).app;
+
+let registrar = app.parameterRegistrar;
```

Comment:
May I propose some name changes (at least on the Wing side):

```js
app.parameters.addSchema(...);
app.parameters.getValue("foo");
```

Reply:
Yea honestly I think these are just good name changes even in the typescript side. Naming is hard